xcdat/dataset.py

```diff
@@ -0,0 +1,43 @@
+"""Dataset module for Dataset related functions."""
+import xarray as xr
+
+from xcdat import axis  # noqa: F401
+
+
+def open_dataset(path: str) -> xr.Dataset:
+    """Wrapper for xarray's ``open_dataset()`` to apply common operations.
+
+    Operations include:
+
+    - Generating axis bounds if they don't exist
+
+    Parameters
+    ----------
+    path : str
+        Path to Dataset.
+
+    Returns
+    -------
+    xr.Dataset
+        Dataset after applying operations.
+
+    Notes
+    -----
+    xarray's ``open_dataset`` opens the file with read-only access. When you
+    modify values of a Dataset, even one linked to files on disk, only the
+    in-memory copy you are manipulating in xarray is modified: the original file
+    on disk is never touched.
+
+    Examples
+    --------
+    Import and call module:
+
+    >>> from xcdat import open_dataset
+    >>> ds = open_dataset("file_path")
+    """
```

Comment:
Example usage for `open_dataset()`. Viewable in the docs [here](https://xcdat.readthedocs.io/en/feature-80-open-dataset/generated/xcdat.dataset.html).